lib/database.js

```diff
@@ -0,0 +1,32 @@
+const { MongoClient } = require("mongodb");
+let client = null;
+let db = null;
+
+async function initDatabase(url, dbName) {
+  client = new MongoClient(url, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  });
+  await client.connect();
+  db = client.db(dbName);
+}
+
+async function closeDatabase() {
+  await client.close();
+}
+
+async function getCollection(collectionName) {
+  try {
+    if (!db) {
+      await initDatabase();
+    }
+    return db.collection(collectionName);
+  } catch (error) {
+    console.error(`You have to initialize the database first: ${error}`);
+    return console.error("Error");
```

Comment:
You have two `console.error` here. 
If this error happens, it doesn't make sure to continue running your webserver.
I would throw that error and do not catch it to force a server failure.
```suggestion
    throw new Error(`You have to initialize the database first: ${error.message}`);
```